### DISCUSSION TITLE
Assistance with Step-by-Step Installation of BackupSheep on VPS

### DISCUSSION THREAD

**Discussion by farlees** [bilal414/backupsheep#5] [Q&A]
Hi Bilal,

I am a satisfied customer of BackupSheep and have been thoroughly impressed with the progress and features of your service. I am now looking to deploy BackupSheep on my own VPS and would greatly appreciate detailed guidance on the installation process.

Could you please provide a comprehensive, step-by-step guide to installing and setting up BackupSheep on a VPS? Specifically, I am interested in:

1. Prerequisites and initial setup steps.
2. Detailed installation instructions.
3. Configuration settings required for optimal performance.
4. Any additional steps or considerations for maintaining the installation.

Could you also please confirm if BackupSheep can be deployed using Vercel or Coolify? If so, I would greatly appreciate detailed instructions on how to proceed with the deployment.

Thank you for your assistance and for providing such a valuable service.

**Comment by djek123** [comment]
Any update on this topic ? I am also a previous backup sheep user and would love to get this going again.

**Comment by Sufiyanshaikh123** [comment]
It will be really helpful if Bilal can provide steps to get it running.
At least a basic guide to get it running

**Comment by dimasrizqi** [comment]
I create step by step instructions for running the application , 
[`backupsheep-installation`](https://dimasrizqi.medium.com/backup-sheep-installation-i-guess-v1-2f7cbb653049)
but im sory can't running full feature because still waiting seeder from backupsheep.

![WhatsApp Image 2024-09-19 at 14 50 25](https://github.com/user-attachments/assets/05fddeb5-f2d2-4af0-8dc9-5fb887d5904b)

**Reply by Sufiyanshaikh123** [reply]
Can you try now?
